doc/telemetry-dev-env/Readme.md

```diff
@@ -0,0 +1,47 @@
+Goto doc/telemetry-dev-env
+Build the image:
+```
+docker build -t telemetry-dev .
+```
+
+Remove container if it exist already:
+```
+docker rm telemetry-dev-env
+```
+Create personal fork repo from sonic-telemetry from git
+
+Create directory '~/src' and Clone git repos in it:
+```
+git clone https://<your_github_username>@github.com/<your_github_username>/sonic-telemetry.git 
+git clone https://<your_github_username>@github.com/<your_github_username>/sonic-mgmt-common.git
+```
+
+Create a new container using the image:
+```
+docker run -it -v ~/src:/src --net=host --name telemetry-dev-env telemetry-dev
+```
+
+Build and start service (Inside container):
+```bash
+
+# Build sources
+cd /src/sonic-mgmt-common && make
+cd /src/sonic-telemetry && make all
+
+# Bootstrap DB.
+mkdir -p /var/run/redis/sonic-db && chown redis  /var/run/redis/sonic-db && install testdata/database_config.json -t /var/run/redis/sonic-db
```

Comment:
could you please explain what does "install testdata/database_config.json -t /var/run/redis/sonic-db" do?

Reply:
it just initializes redis with different DBs and its number . Content found here https://github.com/Azure/sonic-telemetry/blob/master/testdata/database_config.json